CMakeLists.txt

```diff
@@ -5,7 +5,10 @@
 cmake_minimum_required(VERSION 3.1)
 project(TYPE_SAFE)
 
-include(external/external.cmake)
+find_package(debug_assert)
+if(NOT debug_assert_FOUND)
+    include(external/external.cmake)
```

Comment:
I'd rather have the find_package() logic in external itself, in case I want to add other dependencies.